Improve the text

pieterhijma opened the pull request:
The changes are mostly rephrases to make the proposal easier to read.

pieterhijma committed:
Improve the text

The changes are mostly rephrases to make the proposal easier to read.

kadet1090 commented on FEPs/FEP-0001-process.md:
```diff
@@ -3,22 +3,22 @@
-| Description    | Definition of new FreeCAD Enhancement Process |
+| Description    | Definition of the FreeCAD Enhancement Process |
 | Status         | Draft                                         |
 | Author(s)      | Kacper Donat (@kadet1090)                     |
```
Can you add yourself as co author?

pieterhijma replied:
Oh thanks! I'm not sure if my contribution justifies co-authorship. I'll add a separate commit as a co-author and then you can decide to add or not.

kadet1090 commented on FEPs/FEP-0001-process.md:
```diff
@@ -38,44 +38,49 @@ such matters and making the decision on the way to proceed.
 
-Everyone! Though given that it is very technical document related to the FreeCAD code and/or
-development process it is mostly aimed for being created by the developer community.
+Anyone!  However, given that FEPs are highly technical documents related to the FreeCAD code and/or
+the development process, it is mostly aimed for being created by the developer community.  Although
+not strictly necessary, it is advised that non-developers cooperate with a FreeCAD developer.
```
I think that we in the end should make it into strict requirement. Maybe something like that (although probably with better language):
```suggestion
FEPs are highly technical documents related to the FreeCAD code and/or
the development process. Therefore they are mostly aimed for being created by the developer community. Anyone from the GitHub Developers Group can make FEP. If member outside of said group would like to propose FEP it must be done in cooperation with some member of it.
```

pieterhijma replied:
Ok, agreed.

pieterhijma committed:
Add myself as co-author

This is on request from Kacper.  I'm not so sure if my contribution
justifies co-authorship, so please feel free to reconsider and decide
otherwise.

pieterhijma committed:
Make required that non-devs collaborate with devs

For non-developers creating a FEP, it is required to work with a member
from the Developers Group.
